src/actions/login.actions.ts

```diff
@@ -11,6 +12,8 @@ class LoginActions extends BaseActions {
     this.loginPage = new LoginPage(page, context);
   }
 
+  url = 'https://practice.automationtesting.in/my-account/';
```

Comment:
Add `https://practice.automationtesting.in/` in param baseUrl in playwright.config.ts. Remove this value